spec/lib/options.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# This function is used to ensure that every config option
+# has a :type and :description key, as they are required for
+# generating the JSON schemas and validating config.
+def assert_type_description(definitions)
```

Comment:
This should follow the same file structure as the rest of `spec/lib` (specifically it should be in `spec/lib/bolt_spec`) and it should be inside a `module` that is included by each test case that needs this method, rather than defining it globally.